### ISSUE TITLE
Test set being used as val while training LSTM model

### ISSUE THREAD

**Issue by thampiman** [thampiman/DeepFin-Series-JPMorgan#4]
It's better to split the train set into train/val and use this val set for model evaluation and hyperparameter tuning. Then retrain on whole set using optimum hyper params.
